### ISSUE TITLE
paymentRequestWithCardForm Crashing App

### ISSUE THREAD

**Issue by jaysig** [tipsi/tipsi-stripe#82]
I am running into an issue when I try to call paymentRequestWithCardForm. I get the following error, which crashes my app.
```
'*** -[__NSCFConstantString stringByAppendingString:]: nil argument' was thrown while invoking paymentRequestWithCardForm on target TPSStripeManager with params (
        {
        requiredBillingAddressFields = full;
        smsAutofillDisabled = 1;
        theme =         {
        };
    },
    70,
    71
)
```
Function Invocation 
```
handleCardPayPress = async () => {
      try {
        this.setState({
          loading: true,
          token: null,
        })
        console.log(stripe,'inner stripe');
        const token = stripe.paymentRequestWithCardForm({
          // Only iOS support this options
          smsAutofillDisabled: true,
          requiredBillingAddressFields: 'full',
          prefilledInformation: {
            billingAddress: {
              name: 'Gunilla Haugeh',
              line1: 'Canary Place',
              line2: '3',
              city: 'Macon',
              state: 'Georgia',
              country: 'US',
              postalCode: '31217',
              email: 'ghaugeh0@printfriendly.com',
            },
          },
        })

```
How I call it

```
                <ARButton
                    style={[styles.changeBtn,styles.btn]}
                    title="CHANGE PAYMENT INFORMATION"
                    type={2}
                    onPress={this.handleCardPayPress}
                    // onPress={()=>this.props.navigation.navigate('PaymentInfo')}
                />
```

**Comment by itsmepetrov** [comment]
@dmytrozavgorodniy please take a look

**Comment by pongponglau** [comment]
I encounter the same issue in iphone 7. However, there is no problem in simulator.

Thanks for your advice.

**Comment by dmytrozavgorodniy** [comment]
Hi everyone! Sorry for late response.
@jaysig, @pongponglau 
Could you please provide more information about your environment:
What version of Stripe framework you are using in our app? (you can check Stripe pod version in your Podfile)
Which version of iOS you are using for testing?

Also, maybe you can provide, what information you are input in form.

**Comment by pongponglau** [comment]
I remove the card preset json information when calling paymentRequestWithCardForm, then the problem is solved. If not, something like listview index out of range error will return.

Cheers

**Comment by jaysig** [comment]
Hi @dmytrozavgorodniy,
I'm using Stripe 10.1.0
I am just running it on the simulator; so, the latest version of iOS

I can not even get the form to render, it just crashes when I try to initialize it. I ended up just using PaymentCardTextField and that worked fine.

**Comment by viczzh** [comment]
Having the same issue. Any solutions?

**Comment by isnifer** [comment]
@viczzh it would be nice if you can create small repo that reproduces this error

**Comment by Uki19** [comment]
I had this issue, the problem was that I forgot to set publishableKey. Don't forget to add 
`stripe.init({
    publishableKey: 'pk_test_UnK1dsvDzXfiGUwHBRFnOHxU',
});`

**Comment by isnifer** [comment]
@jaysig @viczzh guys did you solve this issue?

**Comment by isnifer** [comment]
Fixed in #109
